### PR TITLE
[ Fabric ] Network deployment verification docs

### PR DESCRIPTION
**Pull request by himanshulalarya**
**Changelog**
- Added Fabric Verification Readme Added
 

**Reviewed by**
@sownak  @suvajit-sarkar @jagpreetsinghsasan 

 

**Linked issue**
#882

### CODE REVIEW

File: docs/source/operations/fabric_verify.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+## DLT Platform 
+
+### Common Components check
+
+#### Common components check ( like flux, namespaces, service accounts, storage classes)
```
Comment (sownak) [inline docs/source/operations/fabric_verify.md:5]: why is the name repeated?

File: docs/source/operations/fabric_verify.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+- Check for pods. Get the list of pods
```
Comment (sownak) [inline docs/source/operations/fabric_verify.md:7]: this is not "check for ..." it is "get list of pods"

File: docs/source/operations/fabric_verify.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+        
+- To describe the pod
+
+        kubectl describe <pod name> -n <namespace>
+-To check the logs of a particular container inside the pod.
+
+        kubectl logs -c <container name in pod> <pod name> -n <namespace>
+
+
+**NOTE:** The pods and other components take some time to setup.You can check the components while the retries occours.
```
Comment (sownak) [inline docs/source/operations/fabric_verify.md:28]: spell check

File: docs/source/operations/fabric_verify.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+#### Platform specific check
+
+##### Fabric ( Specific components check like ca tools, ca server, orderer n peer)
+
+Stage 1.
```
Comment (sownak) [inline docs/source/operations/fabric_verify.md:34]: Aren't these checks common? Stage 1 needs to be a list.

File: docs/source/operations/fabric_verify.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+Check for flux pods using, kubectl get pods -A
+
+    kubectl edit deployment/flux-{{ network.env.type }}
```
Comment (sownak) [inline docs/source/operations/fabric_verify.md:37]: provide example

File: docs/source/operations/fabric_verify.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+
+    kubectl logs -f <flux pod name>
+
+Possible errors, if namespaces, storege classes etc are not generated.
+Value files are not properly generated or not properly pushed.
+- Check whether git branch is right.
+- Check whether the git credentials are right.
+
+Stage 3. 
+Check for ca-server.
```
Comment (sownak) [inline docs/source/operations/fabric_verify.md:52]: This is where fabric section begins

File: docs/source/operations/fabric_verify.md [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+Check if the pod is up and running.
+If not, Check flux-helm operator  logs..
+Possible errors
+- Value files are not generated right. Release failed
```
Comment (sownak) [inline docs/source/operations/fabric_verify.md:56]: Add how to check helmrelease errors(describe helmrelease). then how to rerun by deleting helmrelease

File: docs/source/operations/fabric_verify.md [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+      localmspid: <msp id of peer>
+      tlsstatus: true
+      address: <peer address>
+    orderer:
+      address: <orderer address>
+
+2. To install the cli.
+
+
+    helm install -f cli.yaml <chart path> -n <namespace>
```
Comment (sownak) [inline docs/source/operations/fabric_verify.md:145]: provide the chart path

File: docs/source/operations/fabric_verify.md [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+Stage 5.
+Check for peers
+Check if the pod is up and running.
+If not, Check flux-helm operator logs..
+Possible errors
+- Value files are not generated right. Release failed
+For further details check the pod logs or describe the helm release files
+
+Stage 6.
+Create channel/ Join channel
```
Comment (jagpreetsinghsasan) [inline docs/source/operations/fabric_verify.md:74]: Also mention the error related to invalid orderer tls certificate (which we fetch from network.yaml's orderer section)

File: docs/source/operations/fabric_verify.md [hunk 10]
```diff
@@ -0,0 +1,167 @@
...
+      address: <peer address>
+    orderer:
+      address: <orderer address>
+
+2. To install the cli.
+
+
+    helm install -f cli.yaml <chart path> -n <namespace>
+
+3. Get the cli pod.
```
Comment (jagpreetsinghsasan) [inline docs/source/operations/fabric_verify.md:147]: Use this command to get the CLI pod container instead
export CLI=$(kubectl get po -n ${ORG1_NS} | grep "cli" | awk '{print $1}')

File: docs/source/operations/fabric_verify.md [hunk 11]
```diff
@@ -0,0 +1,167 @@
...
+
+    
+    kubectl get pod -n <namespace>
+
+4. Copy the cli pod name from the output list and enter the cli using.
+
+    
+    kubectl exec -it <cli pod name> -n <namespace> -- bash
```
Comment (jagpreetsinghsasan) [inline docs/source/operations/fabric_verify.md:155]: Change this to
kubectl exec -it $CLI -n <namespace> -- bash

File: docs/source/operations/fabric_verify.md [hunk 12]
```diff
@@ -0,0 +1,167 @@
...
+    
+5. To see which chaincodes are installed 
+
+
+    peer chaincode list --installed
+
```
Comment (jagpreetsinghsasan) [inline docs/source/operations/fabric_verify.md:161]: Also add the logic to check if the chaincode is instantiated or not
peer chaincode list --instantiated -C allchannel

**Pull request closed without merging**